### PR TITLE
update (divide customMarker as customMarkerLeft and customMarkerRight)

### PR DESCRIPTION
**Pull request by sadeghipour**
This update provides two configurable marker as left and right.

### CODE REVIEW

**Comment by ptomasroos** [comment]
It would be great if you could make this backwards compatible? Now it will break for people using customMarker.

**Comment by ptomasroos** [comment]
Impossible for me to track the changes when you've changed over 500 lines

**Comment by sadeghipour** [comment]
just track
TODO:Here added by sadeghipour

**Comment by sadeghipour** [comment]
sorry for auto code reformat

**Comment by ptomasroos** [comment]
Please remove it from formatting if I’m gonna review this

**Comment by sadeghipour** [comment]
updated

File: MultiSlider.js [hunk 1]
```diff
@@ -70,6 +74,10 @@ export default class MultiSlider extends React.Component {
       slipDisplacement: 200,
     },
     customMarker: DefaultMarker,
+
+      customMarkerLeft: DefaultMarker,
```
Comment (ptomasroos) [inline MultiSlider.js]: wrong indentation here

File: MultiSlider.js [hunk 2]
```diff
@@ -336,6 +344,11 @@ export default class MultiSlider extends React.Component {
       ? selectedStyle || styles.selectedTrack
       : unselectedStyle;
     const Marker = this.props.customMarker;
+
+      const MarkerLeft = this.props.customMarkerLeft;
+      const MarkerRight = this.props.customMarkerRight;
```
Comment (ptomasroos) [inline MultiSlider.js]: wrong indentation here

File: MultiSlider.js [hunk 3]
```diff
@@ -410,15 +439,30 @@ export default class MultiSlider extends React.Component {
...
-              <Marker
-                pressed={this.state.twoPressed}
-                markerStyle={this.props.markerStyle}
-                pressedMarkerStyle={this.props.pressedMarkerStyle}
-                currentValue={this.state.valueTwo}
-                enabled={this.props.enabledTwo}
-                valuePrefix={this.props.valuePrefix}
-                valueSuffix={this.props.valueSuffix}
-              />
+                {isMarkersSeparated===false &&
```
Comment (ptomasroos) [inline MultiSlider.js]: spaces around ===

File: MultiSlider.js (hunk 1 above)
Comment (sadeghipour) [reply MultiSlider.js]: Updated in last commit:
https://github.com/sadeghipour/react-native-multi-slider/blob/master/MultiSlider.js#L78

File: MultiSlider.js (hunk 2 above)
Comment (sadeghipour) [reply MultiSlider.js]: Updated in last commit:
https://github.com/sadeghipour/react-native-multi-slider/blob/master/MultiSlider.js#L348

File: MultiSlider.js (hunk 3 above)
Comment (sadeghipour) [reply MultiSlider.js]: Updated in last commit:
https://github.com/sadeghipour/react-native-multi-slider/blob/master/MultiSlider.js#L442

**Comment by ptomasroos** [comment]
Thanks for the fixes, can you also update the example app to showcase this so we at least have _some_ documentation on it. I'm ready to merge this afterwards!

**Comment by sadeghipour** [comment]
doc added but I'm not sure that is clear enough.

File: MultiSlider.js [hunk 4]
```diff
@@ -410,15 +438,29 @@ export default class MultiSlider extends React.Component {
...
-              <Marker
-                pressed={this.state.twoPressed}
-                markerStyle={this.props.markerStyle}
-                pressedMarkerStyle={this.props.pressedMarkerStyle}
-                currentValue={this.state.valueTwo}
-                enabled={this.props.enabledTwo}
-                valuePrefix={this.props.valuePrefix}
-                valueSuffix={this.props.valueSuffix}
-              />
+                {isMarkersSeparated === false &&
```
Comment (ptomasroos) [inline MultiSlider.js]: How about changing this to ? : syntax instead since its a if else. Same for MarkerLeft

**Pull request merged**